src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/SentencesControllerTest.kt

```diff
@@ -77,15 +110,27 @@ internal class SentencesControllerTest(
           [
             {
               "dateOfSentencing": "1990-01-01",
-              "isActive": true
+              "isActive": true,
+              "terms": [
+                { "years": 10, "months": null, "weeks": 5, "days": 5 },
+                { "years": 25, "months": null, "weeks": 5, "days": 4 }
+              ]
             },
             {
               "dateOfSentencing": "1991-01-01",
-              "isActive": false
+              "isActive": false,
+              "terms": [
+                { "years": 15, "months": 6, "weeks": 2, "days": null },
+                { "years": null, "months": 6, "weeks": 2, "days": 5 }
+              ]
             },
             {
               "dateOfSentencing": "1992-01-01",
-              "isActive": null
+              "isActive": null,
+              "terms": [
+                { "years": 15, "months": null, "weeks": null, "days": null },
+                { "years": null, "months": null, "weeks": 2, "days": null }
```

Comment:
Pointing out that this is NOT how the data is exposed to us from upstream - there are never null fields being sent to us, they are either missing or populated.